### DISCUSSION TITLE
Can I just copy the trilium data folder between different computers and sync?

### DISCUSSION THREAD

**Discussion by gerroon** [TriliumNext/Trilium#2740] [Q&A] [answered]
Hi

I do not want to waste time with setting up Trilium from scratch when I am going to install it on a new device. can I just copy the trilium data folder to the new device? Would that be an issue with the actual sync?

**Comment by XXXJetfireXXX** [comment, marked as answer]
Sure, you can copy the data folder to a new device and it should work fine. However, if you're gonna sync that device later with the same server instance - a conflict might appear: https://github.com/zadam/trilium/wiki/Synchronization
![image](https://user-images.githubusercontent.com/43646178/159598438-c214823b-1d68-4cb5-93c4-edadd30f147e.png)

**Comment by gerroon** [comment]
Thanks for the reply. Are you saying that all files are going to conflict due to the copied database? If conflicts might happen than this contradicts "it should work fine" line.

**Reply by XXXJetfireXXX** [reply]
Not all files, just the ones that are different from each other in different instances. As it says, the newer change is gonna be picked automatically, discarding the older change. That means, if you change something on that device and then decide to sync it - a new change is gonna be applied to the server on top of the old one, but you still can recover the older version of a note through *note revisions*.

**Reply by XXXJetfireXXX** [reply]
Though, I'd still recommend doing a backup first before trying to do anything.

**Reply by XXXJetfireXXX** [reply]
To be honest, @zadam would answer this question far more precisely than me :)

**Comment by zadam** [comment]
It's not clear from your description, but I assume you already have sync setup (client + server).

It's perfectly safe to copy client's `trilium-data` to other clients and in such a case there shouldn't be any conflict arising.

What @XXXJetfireXXX mentioned would happen if you didn't have sync in the first place and just copied `trilium-data` to another PC, then used them side by side for a considerable time and only then tried to set up a sync. Then there would be conflicts in notes which were edited in the "no-sync" period.

**Reply by gerroon** [reply]
Thanks Zadam. That helps a lot.
